build.gradle.kts

```diff
@@ -220,29 +221,45 @@ tasks.remapSourcesJar {
 /* Bypassing https://github.com/johnrengelman/shadow/issues/111 */
 // Use Zip instead of Jar as to not include META-INF
 val kotlinDependencyCollectionJar by tasks.creating(Zip::class) {
-		archiveFileName.set("kotlin-libraries-wrapped.jar")
-		destinationDirectory.set(project.layout.buildDirectory.dir("kotlinwrapper"))
-		from(kotlinDependencies)
-		into("neu-kotlin-libraries-wrapped")
+	archiveFileName.set("kotlin-libraries-wrapped.jar")
+	destinationDirectory.set(project.layout.buildDirectory.dir("kotlinwrapper"))
+	from(kotlinDependencies)
+	into("neu-kotlin-libraries-wrapped")
+}
+
+tasks.register("updateBackupRepo") {
+	doLast {
+		val url = URL("https://github.com/NotEnoughUpdates/NotEnoughUpdates-REPO/archive/refs/heads/master.zip")
+		val destination = File(
+			project.layout.projectDirectory.asFile.absolutePath
+				.plus("/src/main/resources/assets/notenoughupdates/repo.zip")
+		)
```

Comment:
You can use .resolve() and you should probably either put it in `build/` or at least .gitignore this.